distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -342,3 +342,9 @@ type Date
     compare_to that =
         sign = self.internal_local_date.compareTo that.internal_local_date
         Ordering.from_sign sign
+
+    ## Compares two Dates for equality.
+    == : Date -> Boolean
+    == that = case that of
+        Date _ -> self.internal_local_date.equals that.internal_local_date
```

Comment:
Before [f16d07e](https://github.com/enso-org/enso/pull/3559/commits/f16d07e640c12721eb329a797a19c7b46bbf3fbc) the `that` argument wasn't `Date` atom and that resulted in `False` return value.